### ISSUE TITLE
Feature crashing

### ISSUE THREAD

**Issue by Kenix97** [xardch/sensum#2]
When Sound ESP is enabled, it crashes the game.

**Comment by Hipisuaw** [comment]
yeah, its true
just delete this

**Comment by Hipisuaw** [comment]
yeah, its true

**Comment by xardch** [comment]
Yes, this is my mistake and it does not work as it should. You can change this by collecting information about steps in a safe thread (at the moment it is called in the DX thread) and continue drawing it in the DX thread on the example of Visuals/ESP.

Or just don't use it.
